specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/virtualWan.json

```diff
@@ -4638,13 +4768,6 @@
           "$ref": "./network.json#/definitions/SubResource",
           "description": "The securityPartnerProvider associated with this VirtualHub."
         },
-        "virtualNetworkConnections": {
```

Comment:
Just curious about this part. Does this represent that virtualNetworkConnection is an independent resource and not linked to virtual hub anymore?

Reply:
The virtualNetworkConnection will still be a child resource of the virtual hub, but it will be independently updated so that operations on the hub doesn't fate share any failures with the virtualNetworkConnection. This also handles other scenarios where the virtualNetworkConnection  could potentially need RBAC access to multiple subscriptions to create cross-subscription virtual network peerings. Creating this hub is simpler w.r.t to this.

Reply:
Removing this from a stable API-version is a breaking change. Will this continue to work in older api-versions?

Reply:
WHat would happen if the virtualHub is created using 2020-05-01 api-version and GET using the prior API-version. What would happen if created using prior api-version and GET using the new one? @shyamshd ?